docs/source/array-examples.rst

```diff
@@ -0,0 +1,9 @@
+Examples
+========
+
+.. toctree::
+   :maxdepth: 1
+
+   examples/array-random.rst
+   examples/array-numpy.rst
+   examples/array-pickle.rst
```

Comment:
What are our objectives for these examples?  Something they can run immediately?  Something that shows off how they would use dask.array in the real world?